tests/conftest.py

```diff
@@ -45,10 +48,28 @@ def mock_butler() -> Iterator[MockButler]:
     yield from patch_butler()
 
 
-@pytest.fixture
+@pytest.fixture(scope="function")
+def aws_credentials() -> None:
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+    os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
```

Comment:
This is a little dangerous because the environment variable settings are process-global and thus leak into the next test because nothing undoes them. Usually when I change the environment, I make it a yield fixture and then set the environment variables back at the end. (I wonder if Safir should have a standard fixture that does this to make it a bit less tedious.)